docs/deployed-contracts/holesky.md

```diff
@@ -10,9 +10,7 @@ We're excited to announce that the deployment campaign for Lido on Holešky is c
 Please be aware that you might encounter unexpected behavior during this period. We're working diligently to ensure a smooth deployment, but unforeseen issues can arise.
 :::
 
-## Holešky testnet
-
-### Lido V2 Core Protocol
+## Lido V2 Core Protocol
```

Comment:
Maybe we can omit 'Lido V2' prefixes here and there (I know it affects not only Holešky :smile: )

Reply:
I would remove this synchronously with the mainnet page to maintain consistency